contracts/test/TestStrategy.sol

```diff
@@ -87,4 +83,8 @@ contract TestStrategy is BaseStrategy {
     function prepareMigration(address _newStrategy) internal override {
         want.transfer(_newStrategy, want.balanceOf(address(this)));
     }
+
+    function protectedTokens() internal override view returns (address[] memory) {
```

Comment:
why didn't you copy the exact method and body from the previous implementation?
Most strategists are going to copy and paste code from `TestStrategy` I would leave here the complete method.

Reply:
I removed `want` from there and made it automatically checked for